src/core/request-body-search.js

```diff
@@ -114,6 +114,34 @@ class RequestBodySearch {
     }
 
     /**
+     * Sets multiple nested aggregation items.
+     * Alias for method `aggregations`
+     *
+     * @param {Array} aggs Array of valid {@link Aggregation} items
+     * @returns {Aggregation} returns `this` so that calls can be chained.
+     * @throws {TypeError} If `aggs` is not an instance of `Array`
+     */
+    aggs(aggs) {
+        return this.aggregations(aggs);
+    }
+
+    /**
+     * Sets multiple nested aggregation items.
+     * This method accepts an array to set multiple nested aggregations in one call.
+     *
+     * @param {Array} aggs Array of valid {@link Aggregation} items
+     * @returns {Aggregation} returns `this` so that calls can be chained.
+     * @throws {TypeError} If `aggs` is not an instance of `Array`
```

Comment:
Could you add another `@throws` annotation? It would throw if any of the items in the array is _not_ an instance of `Aggregation`.